feat(skeletonTable): Added skeleton table component. (#56)
patternfly/react-component-groups: packages/module/patternfly-docs/content/extensions/component-groups/examples/Skeleton/Skeleton.md

---
# Sidenav top-level section
# should be the same for all markdown files
section: extensions
subsection: Component groups
# Sidenav secondary level section
# should be the same for all markdown files
id: Skeleton table
# Tab (react | react-demos | html | html-demos | design-guidelines | accessibility)
source: react
# If you use typescript, the name of the interface to display props for
# These are found through the sourceProps function provided in patternfly-docs.source.js
propComponents: ['SkeletonTable']
---
import SkeletonTable from '@patternfly/react-component-groups/dist/dynamic/SkeletonTable';

The skeleton table is used to handle the loading of a table.  It indicates data for each of the rows are loading by
using the [skelton](https://www.patternfly.org/components/skeleton) component in each cell. Once the data is loaded, the
skeleton table should be replaced with the table component containing the real data.

## Examples

### Basic skeleton table

> The following is a basic example of a skeleton table.

Reviewer: ```suggestion
To indicate that a table's cells are still loading, a basic skeleton table uses the [skeleton](https://www.patternfly.org/components/skeleton) component to place a placeholder skeleton in each cell. Once the data is loaded, the skeleton table is replaced with a table containing the real data.```

> To indicate that a table's cells are still loading, a basic skeleton table uses the [skeleton](https://www.patternfly.org/components/skeleton) component to place a placeholder skeleton in each cell. Once the data is loaded, the skeleton table is replaced with a table containing the real data.```

Reviewer 2: ```suggestion
To indicate that a table's cells are still loading, a basic skeleton table uses the [skeleton](https://www.patternfly.org/components/skeleton) component to place a placeholder skeleton in each cell. Once the data is loaded, the skeleton table is replaced with a table containing the real data.
```

```js file="./SkeletonTableExample.tsx"

```

> ### Skelton Table loading and replaced with table component
> 
> This example shows how to use the skeleton table to indicate the data is loading, and then replace it with the table component once the data is available.
> To simulate the loading of the table use the `Load Table` button below the table.

Reviewer: ```suggestion
### Full loading simulation

The following example demonstrates the typical behavior of a skeleton table transitioning to a normal table as the data becomes available.

To simulate this loading process, select the `Load Table` button and wait for the data to populate.

```

> To simulate this loading process, select the `Load Table` button and wait for the data to populate.

Reviewer 2: ```suggestion
To simulate this loading process, select the `Reload table` button and wait for the data to populate.
```
